src/app/main/component/user/components/profile/edit-profile/edit-profile.component.scss

```diff
@@ -98,12 +98,12 @@ input:checked ~ .checkmark::after {
 
       input.ng-invalid.ng-touched,
       textarea.ng-invalid.ng-touched {
-        border: 1px solid red !important;
+        border: 1px solid #eb180d !important;
```

Comment:
It's recommended to use variables for colors from _colorsUbs.scss 
For this case is suitable --ubs-primary-red: #eb180d;

Reply:
I applied variable for color: var(--error-red) from _colors.scss